docker/precompiled/Dockerfile

```diff
@@ -21,7 +21,7 @@ RUN julia -e 'using Pkg; \
     Pkg.add(["Pluto", "PlutoUI", "Plots", "PackageCompiler"]); \
 	Pkg.precompile(); \
     using PackageCompiler; \
-    create_sysimage([:Pluto, :PlutoUI]; precompile_execution_file="warmup.jl", replace_default=true)' \
+    create_sysimage([:Pluto, :PlutoUI]; precompile_execution_file="warmup.jl", replace_default=true, cpu_target = PackageCompiler.default_app_cpu_target())' \
```

Comment:
Include Plots while creating the sysimage; it was the one creating the issue

Reply:
For me, the portability issue was also present after removing Plots. But I'll check if it still works after re-adding Plots.